src/SolutionTemplate/UnoSolutionTemplate.VISX/SetGlobalGlobalPropertiesForUno.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.Globalization;
+using Microsoft.VisualStudio.ProjectSystem;
+using Microsoft.VisualStudio.ProjectSystem.Build;
+using Task = System.Threading.Tasks.Task;
+
+namespace UnoSolutionTemplate
+{
+	// https://github.com/microsoft/VSProjectSystem/blob/4ad5716f8fca76403699b818c3d907ce8a8b9a38/doc/extensibility/IProjectGlobalPropertiesProvider.md
+	[ExportBuildGlobalPropertiesProvider]
+	[AppliesTo("")]
+	public class SetGlobalGlobalPropertiesForUno : StaticGlobalPropertiesProviderBase
```

Comment:
Global**Global**Properties?

... they are globally global, in a global way?

Reply:
Universally local globals to the current build :)